public/src/client/topic.js

```diff
@@ -143,7 +143,7 @@ define('forum/topic', [
 			if (components.get('post/anchor', postIndex).length) {
 				return navigator.scrollToPostIndex(postIndex, true);
 			}
-		} else if (bookmark && (!config.usePagination || (config.usePagination && ajaxify.data.pagination.currentPage === 1)) && ajaxify.data.postcount > 5) {
+		} else if (bookmark && (!config.usePagination || (config.usePagination && ajaxify.data.pagination.currentPage === 1)) && ajaxify.data.postcount > ajaxify.data.config.bookmarkThreshold) {
```

Comment:
If `bookmarkThreshold` is only used on the topic page there is no need to put it in the global config object. It should be just in `ajaxify.data.bookmarkThreshold`